torch/csrc/jit/script/init.cpp

```diff
@@ -1010,6 +1010,13 @@ void initJitScriptBindings(PyObject* module) {
             }
             return result;
           })
+      .def(
+          "__enable_profiling__",
```

Comment:
`__` is reserved for magic methods, which this is not. Also, this is threaded correctly because it is only defined on Function, but we also have Modules and Methods which can recursively create executors. Rather than thread a boolean through, I'd prefer this just be a thread_local property that resource guards sets in python. See enable_cpu_fuser for an example. In the end this won't be a user-exposed option anyway, so a light touch on where we add APIs is best.